src/styles/block/watch.scss

```diff
@@ -0,0 +1,60 @@
+@import "../utitlis/variables";
+
+.watch {
+  width: $size-watch;
+  height: $size-watch;
+  border: 20px solid $border-watch;
+  border-radius: $radius;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  position: relative;
+
+  &__center {
+    width: $size-center-gyre;
+    height: $size-center-gyre;
+    background-color: $second-color;
+    border-radius: $radius;
```

Comment:
I would create a mixin for making a circle because you've already used circle twice
Like 
```
@mixin circle($width, $color) {
  width: $width;
  height: $width;
  border-radius: 50%;
  background-color: $color;
}
```